src/code/components/chromosome.js

```diff
@@ -25,11 +25,10 @@ import {getChromosomeDescriptor} from '../fv-components/fv-chromosome-image';
  * @param {number} height - the height in pixels of the rendered chromosome
  * @param {object} displayStyle - this style will be applied to the actual chromosome, as opposed to its labels
  */
-const ChromosomeView = ({chromosome, chromosomeDescriptor, ChromosomeImageClass=ChromosomeImageView, userChangeableGenes = [], visibleGenes = [], hiddenAlleles = [], small = false, editable = true, selected = false, onAlleleChange, onChromosomeSelected, showLabels = true, labelsOnRight = true, orgName, height, displayStyle = {}}) => {
+const ChromosomeView = ({chromosome, chromosomeDescriptor, ChromosomeImageClass=ChromosomeImageView, userChangeableGenes = [], visibleGenes = [], hiddenAlleles = [], small = false, editable = true, selected = false, onAlleleChange, onChromosomeSelected, showLabels = true, labelsOnRight = true, empty = false, orgName, height, displayStyle = {}}) => {
```

Comment:
Previously, chromosomes would be rendered as empty and unlabeled if `chromosomeDescriptor` and `chromosome` props were both empty. For the "randomize chromosomes" animation, we'd like to be able to leave the labels in place while the chromosomes flash. So, I added an `empty` prop to `Chromosome`, so you can render it empty, while still passing information on how to label it.